File: CoreList/NewsListCell.m
//
//  NewsListCell.m
//  CoreList
//
//  Created by 成林 on 15/6/5.
//  Copyright (c) 2015年 muxi. All rights reserved.
//

#import "NewsListCell.h"
#import "NewsListModel.h"

@interface NewsListCell ()

@property (weak, nonatomic) IBOutlet UILabel *idLabel;

@property (weak, nonatomic) IBOutlet UILabel *titleLabel;

@property (weak, nonatomic) IBOutlet UILabel *descLabel;

@property (weak, nonatomic) IBOutlet UILabel *contentLabel;





@end



@implementation NewsListCell

-(void)dataFill:(NewsListModel *)newsListModel{
    
    _idLabel.text = [NSString stringWithFormat:@"%@",@(newsListModel.hostID)];
    
    _titleLabel.text = newsListModel.title;
    
    _descLabel.text = newsListModel.about;
    
    _contentLabel.text = newsListModel.content;
}

















@end

File: CoreList/App/News/View/NewsListCell.swift
//
//  NewsListCell.swift
//  CoreList
//
//  Created by 冯成林 on 16/1/3.
//  Copyright © 2016年 muxi. All rights reserved.
//

import UIKit

class NewsListCell: CoreListTableViewCell {

    @IBOutlet weak var idLabel: UILabel!

    @IBOutlet weak var titleLabel: UILabel!
    
    @IBOutlet weak var descLabel: UILabel!
    
    @IBOutlet weak var contentLabel: UILabel!
    
    @IBOutlet weak var typeLabel: UILabel!
    
    static let rid : String = "NewsListCell"

    
}



//extension NewsListCell {
//    
//    
//    
//    override class func dequeReuse(_ tableView: UITableView!) -> CoreListTableViewCell!{
//        
//        var cell: CoreListTableViewCell! = tableView.dequeueReusableCell(withIdentifier: rid) as? CoreListTableViewCell
//        
//        if cell == nil {cell = Bundle.main.loadNibNamed(rid, owner: nil, options: nil)?.first as! CoreListTableViewCell}
//        
//        return cell!
//    }
//    
//    
//    
//    override func dataFill(_ coreModel: CoreModel!) {
//        
//        let model = coreModel as! NewsListModel
//    
//        idLabel.text = "\(model.hostID)"
//        
//        titleLabel.text = model.title;
//
//        contentLabel.text = model.content;
//        
//        typeLabel.text = "\(model.score)"
//        
//    }
//}
